workspace/notebook/py_unit_tests_functions.json

```diff
@@ -881,6 +881,44 @@
 					"    return df.count() == 0"
 				],
 				"execution_count": null
+			},
+			{
+				"cell_type": "code",
+				"metadata": {
+					"jupyter": {
+						"source_hidden": false,
+						"outputs_hidden": false
+					},
+					"nteract": {
+						"transient": {
+							"deleting": false
+						}
+					}
+				},
+				"source": [
+					"from collections import Counter\r\n",
+					"\r\n",
+					"def Array_duplicates(curated_db: str, curated_table: str, sourceColumn: str):\r\n",
```

Comment:
Can we follow the naming convention for naming test functions please? A more relevant name could be `test_duplicates_in_array_column`

Reply:
I renamed function name